src/utilities/findBreakingChanges.js

```diff
@@ -62,50 +62,20 @@ export const DangerousChangeType = Object.freeze({
   ARG_DEFAULT_VALUE_CHANGE: 'ARG_DEFAULT_VALUE_CHANGE',
 });
 
-export type BreakingChange = {
-  type: $Keys<typeof BreakingChangeType>,
-  description: string,
-  ...
-};
-
-export type DangerousChange = {
-  type: $Keys<typeof DangerousChangeType>,
+export type SchemaChange = {
+  type: $Keys<typeof BreakingChangeType> | $Keys<typeof DangerousChangeType>,
```

Comment:
It's better to create common `SchemaChangeType`:
```js
export const SchemaChangeType = Object.freeze({
  ...BreakingChangeType,
  ... DangerousChangeType,
});
```

Reply:
Perfect. I was thinking about it